data/reusables/files/choose-commit-email.md

```diff
@@ -1,4 +1,4 @@
 {% if currentVersion == "free-pro-team@latest" %}
-1. Below the commit message fields, click the email address drop-down menu and choose a Git author email address. Only verified email addresses appear in this drop-down menu. If you enabled email address privacy, then `<username>@users.noreply.github.com` is the default commit author email address.  For more information, see "[Setting your commit email address](/articles/setting-your-commit-email-address)."
+1. If you have more than one email address associated with your GitHub Account can you select the email adress to use as Git author email adress from a drop-down menu. Only verified email addresses appear in this drop-down menu. If you enabled email address privacy, then `<username>@users.noreply.github.com` is the default commit author email address.  For more information, see "[Setting your commit email address](/articles/setting-your-commit-email-address)."
```

Comment:
```suggestion
1. If you have more than one email address associated with your {% data variables.product.product_name %} account, click the email address drop-down menu and select the email address to use as the Git author email address. Only verified email addresses appear in this drop-down menu. If you enabled email address privacy, then `<username>@users.noreply.github.com` is the default commit author email address.  For more information, see "[Setting your commit email address](/articles/setting-your-commit-email-address)."
```

I had a few tweaks and suggestions that I'll explain 😄  instead of typing out GitHub, we like to use the `{% data variables.product.product_name %}` variable, which is a dynamic reusable that will populate with the product that you are using. For instance, this will say GitHub account if a user is reading the GitHub.com version of the article, and GitHub Enterprise Server account if a user is reading the Enterprise version of the article. It makes maintenance much easier! 👍